tests/test_sim.py

```diff
@@ -516,6 +516,51 @@ def process():
                 self.assertEqual((yield self.o), 0)
             sim.add_sync_process(process)
 
+    def setUp_clock_phase(self):
+        self.phase0 = ClockDomain()
+        self.phase90 = ClockDomain()
+        self.phase180 = ClockDomain()
+        self.phase270 = ClockDomain()
+        self.check = ClockDomain()
+
+        self.m = Module()
+        self.m.domains += [
+                self.phase0, self.phase90, self.phase180,
+                self.phase270, self.check
```

Comment:
Should only have one level of indentation here.

Reply:
You can also just remove this block to assign all of the clocks directly to a domain with a name which would work a lot nicer I think.

Reply:
yes, true. I had to look up the syntax on how to get the clock from a domain defined that way =)